services/dy-dash/cc-rabbit-0d/src/cc-rabbit-0d/cc-rabbit-0d.py

```diff
@@ -0,0 +1,557 @@
+        title=yLabel,
+        gridcolor=osparc_style['gridColor']
+    )
+    margin = 10
+    y_label_padding = 50
+    x_label_padding = 30
+    fig['layout']['margin'].update(
+        l=margin+y_label_padding,
+        r=margin,
+        b=margin+x_label_padding,
+        t=margin,
+    )
+
+    fig['layout'].update(
+        autosize=True,
+        height=GRAPH_HEIGHT,
+        showlegend=False,
+        plot_bgcolor=osparc_style['backgroundColor'],
+        paper_bgcolor=osparc_style['backgroundColor'],
+        font=dict(
+            color=osparc_style['color']
+        )
+    )
+    return fig
+
+def get_empty_cols_graphs(labelPairs=[['x', 'y']]):
+    fig = tools.make_subplots(rows=1,
+                            cols=len(labelPairs),
+                            shared_xaxes=True,
+                            vertical_spacing=0.05
+    )
+
+    for idx, labelPair in enumerate(labelPairs):
+        suffix = str(idx)
+        if idx is 0:
+            suffix = ''
+        fig['layout']['xaxis'+suffix].update(
+            title=labelPair[0],
+            gridcolor=osparc_style['gridColor']
+        )
+        fig['layout']['yaxis'+suffix].update(
+            title=labelPair[1],
+            gridcolor=osparc_style['gridColor']
+        )
+    margin = 10
+    y_label_padding = 50
+    x_label_padding = 30
+    fig['layout']['margin'].update(
+        l=margin+y_label_padding,
+        r=margin,
+        b=margin+x_label_padding,
+        t=margin,
+    )
+
+    fig['layout'].update(
+        autosize=True,
+        height=GRAPH_HEIGHT,
+        showlegend=False,
+        plot_bgcolor=osparc_style['backgroundColor'],
+        paper_bgcolor=osparc_style['backgroundColor'],
+        font=dict(
+            color=osparc_style['color']
+        )
+    )
+    return fig
+
+def get_empty_rows_graphs(xLabel='x', yLabels=['y']):
+    fig = tools.make_subplots(rows=len(yLabels),
+                            cols=1,
+                            shared_xaxes=True,
+                            horizontal_spacing=0.05
+    )
+
+    fig['layout']['xaxis'].update(
+        title=xLabel,
+        gridcolor=osparc_style['gridColor']
+    )
+    for idx, yLabel in enumerate(yLabels):
+        suffix = str(idx)
+        if idx is 0:
+            suffix = ''
+        fig['layout']['yaxis'+suffix].update(
+            title=yLabel,
+            gridcolor=osparc_style['gridColor']
+        )
+
+    margin = 10
+    y_label_padding = 50
+    x_label_padding = 30
+    fig['layout']['margin'].update(
+        l=margin+y_label_padding,
+        r=margin,
+        b=margin+x_label_padding,
+        t=margin,
+    )
+
+    fig['layout'].update(
+        autosize=True,
+        height=GRAPH_HEIGHT,
+        showlegend=False,
+        plot_bgcolor=osparc_style['backgroundColor'],
+        paper_bgcolor=osparc_style['backgroundColor'],
+        font=dict(
+            color=osparc_style['color']
+        )
+    )
+    return fig
+
+
```

Comment:
Good practices:
 - use literal constants for labels e.g. ``TIME=time (sec)``
 - a list of ``empty_graph`` instead of a variable per graph? Then, e.g. ``app.layout`` is automatically formed.
 -